enumerables.rb

```diff
@@ -0,0 +1,188 @@
+module Enumerable
+  # 1. Create my_each
+  def my_each
```

Comment:
* [ ] should have working my_each  identical and returns the same thing as ruby's[`each`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). 
([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
- 1
``` p [1,2,3,4,5]my_each``` != ``` p [1,2,3,4,5]each```
``` p [1,2,3,4,5]each``` => should return ```<Enumerator: [1, 2, 3, 4]:each>```
therefore 
```p [1,2,3,4,5]my_each``` => should return``` <Enumerator: [1, 2, 3, 4]:my_each>```
you can handle this by adding this condition to your method
```  return to_enum(the_name_your_method)  unless block_given?```
- 2 

p [1,2,3,4].my_each{|a|a} != p [1,2,3,4].each{|a|a}